[DOCU-2556] Streamlined Gateway Creation in Customer Cloud Provider (#4557)
Kong/docs.konghq.com: app/konnect/runtime-manager/runtime-instances/gateway-runtime-azure.md

---
title: Set up a Kong Gateway Runtime with Azure
no_version: true
content_type: how-to
alpha: true
---

Set up a Azure runtime instance through the
[{{site.konnect_short_name}} Runtime Manager](/konnect/runtime-manager) and
configure your instance to accept configuration from
{{site.konnect_short_name}}. The Runtime Manager keeps track of all runtime
instances associated with the {{site.konnect_saas}} account.

> When you create an Azure runtime instance, runtime manager provides a template for the runtime instance in Azure.

Reviewer: ```suggestion
Runtime Manager provides a pre-populated template for a runtime instance in Azure.
```

> {:.note}
> > **Note:** Kong does not host runtimes. You must install and host your own
> runtime instances.

Reviewer: ```suggestion
```
Same as above.
